contracts/bonding/BondingManager.sol

```diff
@@ -357,11 +354,23 @@ contract BondingManager is IBondingManager, Manager {
 
         // Calculate number of tokens to mint
         uint256 mintedTokens = mintedTokensPerReward(msg.sender);
-        /* // Mint token reward and allocate to this protocol contract */
+        // Mint token reward and allocate to this protocol contract
         token.mint(this, mintedTokens);
 
         // Compute transcoder share of minted tokens
         uint256 transcoderRewardShare = mintedTokens.mul(t.blockRewardCut).div(100);
+
+        delegators[msg.sender].bondedAmount = delegators[msg.sender].bondedAmount.add(transcoderRewardShare);
```

Comment:
So if I'm a delegator, I'm expected to call reward now?

Reply:
No, so now if I am a registered transcoder I might have 2 structs on-chain that store data relevant to me associated with my address. The `Transcoder` struct stores transcoding relevant info i.e. blockRewardCut, feeShare. The `Delegator` struct stores bonded amount, withdraw round, etc. When I call reward, the bonded amount field in the `Delegator` struct associated with my address will be updated. Thus, if I want to unbond and withdraw my bonded stake I do so as a delegator. This is to separate transcoder candidacy and delegation - in this case I am both a transcoder and a delegator (likely delegated to myself).